api/v1/models/status_response.go

```diff
@@ -33,6 +33,9 @@ type StatusResponse struct {
 	// Status of bandwidth manager
 	BandwidthManager *BandwidthManager `json:"bandwidth-manager,omitempty"`
 
+	// Status of BGP
```

Comment:
Please put the changes to the auto-generated file here into the same commit where you edited the `openapi.yaml`.

Also, I think you need to commit the file containing `models.BgpStatus`, it seems to be missing.